cypress/support/step-definitions/common-steps.js

```diff
@@ -35,18 +35,10 @@ const TEXT_ALIGN = 'justify-content';
 const TEXT_ALIGN_START = 'flex-start';
 const TEXT_ALIGN_END = 'flex-end';
 
-Given('I open design systems {word} {string} component page', (type, component) => {
-  visitComponentUrl(component, type, false, 'design-system-');
-});
-
 Given('I open Design Systems {word} {string} component docs page', (type, component) => {
```

Comment:
I think that this step is used only for `duellingPickList` component which is also available in `canvas`, so try to open this component in no iframe. If it'll be possible you can remove this step

Reply:
done :) good spot